src/journey-maps/angular/journey-maps.ts

```diff
@@ -870,10 +878,14 @@ export class SbbJourneyMaps implements OnInit, AfterViewInit, OnDestroy, OnChang
 
     return (
       optionsAmount === 0 ||
-      (optionsAmount === 1 && !this.journeyMapsRoutingOption?.routesMetaInformations) ||
+      (optionsAmount === 1 &&
+        !this.journeyMapsRoutingOption?.routesMetaInformations &&
+        !this.journeyMapsRoutingOption?.journeyMetaInformation) ||
       (optionsAmount === 2 &&
         !!this.journeyMapsRoutingOption?.routes &&
-        !!this.journeyMapsRoutingOption?.routesMetaInformations)
+        !!this.journeyMapsRoutingOption?.routesMetaInformations) ||
+      (!!this.journeyMapsRoutingOption?.journey &&
+        !!this.journeyMapsRoutingOption?.journeyMetaInformation)
```

Comment:
In the last part of the if clause you should also include `optionsAmout === 2` or not?